includes/wikia/services/AvatarService.class.php

```diff
@@ -178,7 +181,7 @@ static function renderAvatar( $userName, $avatarSize = 20 ) {
 			'width' => $avatarSize,
 			'height' => $avatarSize,
 			'class' => 'avatar',
-			'alt' => $userName,
+			'alt' => IP::isIPAddress( $userName ) ? wfMessage( 'oasis-anon-user' )->inContentLanguage()->text() : $userName,
```

Comment:
For anonymous users user language = content language anyways, so this way we spare `wgLang` calculation since content language is already known.